src/platform/ios/ffi.rs

```diff
@@ -64,10 +64,20 @@ extern {
 
 extern {
     pub fn setjmp(env: *mut c_void) -> c_int;
-    pub fn longjmp(env: *mut c_void, val: c_int);
+    pub fn longjmp(env: *mut c_void, val: c_int) -> !;
 }
 
-pub type JmpBuf = [c_int; 27];
+// values taken from "setjmp.h" header in xcode iPhoneOS/iPhoneSimulator SDK
+#[cfg(any(target_arch = "x86_64"))]
+pub const JBLEN: usize = ((9 * 2) + 3 + 16);
+#[cfg(any(target_arch = "x86"))]
+pub const JBLEN: usize = (18);
```

Comment:
What's the significance of the parentheses?

Reply:
I copied it directly from the "setjmp.h" header where they `#define _JBLEN (value)`. The extra parentheses don't make sense in rust :). I'll remove them.